cli/foundation-models/system/inference/text-classification/text-classification-online-endpoint.sh

```diff
@@ -0,0 +1,85 @@
+set -x
+# the commands in this file map to steps in this notebook: https://aka.ms/azureml-infer-online-sdk-text-classification
+# the sample scoring file available in the same folder as the above notebook
+
+# script inputs
+registry_name="azureml"
+subscription_id="<SUBSCRIPTION_ID>"
+resource_group_name="<RESOURCE_GROUP>"
+workspace_name="<WORKSPACE_NAME>"
+
+# This is the model from system registry that needs to be deployed
+model_name="microsoft-deberta-base-mnli"
```

Comment:
This model is from the entailment/contraction text classification type.  Update this model to match the sentiment analysis one from SDK.

Reply:
Fixed